<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лічильник</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.534078, y1:0.192773, x2:0.521725, y2:1, stop:0.254902 rgba(46, 52, 54, 255), stop:0.647059 rgba(44, 0, 60, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="countText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: italic 37pt &quot;Ubuntu Condensed&quot;;
	transform:rotate(90deg);
	background:#ffcc00;
	text-align:center;
	color:rgb(32, 74, 135);
	border-color:rgb(32, 74, 135);
}

QLabel:hover{
	color:rgb(52, 101, 164);
	border-color:rgb(114, 159, 207);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIncrem">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(32, 74, 135);
	border:0px;
	font: 75 italic 100px &quot;Ubuntu Condensed&quot;;
}

QPushButton:hover{
	background-color: rgb(52, 101, 164);
}

QPushButton:pressed{
	background-color: rgb(114, 159, 207);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecrem">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(32, 74, 135);
	border:0px;
	font: 75 italic 100px &quot;Ubuntu Condensed&quot;;
}

QPushButton:hover{
	background-color: rgb(52, 101, 164);
}

QPushButton:pressed{
	background-color: rgb(114, 159, 207);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <zorder>countText</zorder>
   <zorder>btnDecrem</zorder>
   <zorder>btnIncrem</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
